Realm/Tests/RealmTests.mm

```diff
@@ -1871,6 +1871,65 @@ - (void)testWritingCopyUsesWriteTransactionInProgress
     }];
 }
 
+#pragma mark - SeedFilePath
+
+- (void)testSeedFilePath {
+    // Create a seed realm with two objects
+    RLMRealmConfiguration *seedConfig = [RLMRealmConfiguration defaultConfiguration];
+    seedConfig.fileURL = RLMTestRealmURL();
+    RLMRealm *realm = [RLMRealm realmWithConfiguration:seedConfig error:nil];
+    [realm beginWriteTransaction];
+    [IntObject createInRealm: realm withValue:@[@1]];
+    [IntObject createInRealm: realm withValue:@[@2]];
+    [realm commitWriteTransaction];
+
+    NSURL *fileURL = [NSURL fileURLWithPath:RLMRealmPathForFile(@"filename.realm")];
+    assert(![[NSFileManager defaultManager] fileExistsAtPath:fileURL.path]);
+
+    // Open realm at destination, expect the objects from seeded path.
+    RLMRealmConfiguration *configuration = [RLMRealmConfiguration defaultConfiguration];
+    configuration.fileURL = fileURL;
+    configuration.seedFilePath = RLMTestRealmURL();
+    RLMRealm *destinationRealm = [RLMRealm realmWithConfiguration:configuration error:nil];
+    RLMResults *intObjects = [IntObject allObjectsInRealm:destinationRealm];
+    XCTAssertEqual(intObjects.count, 2U, @"Expect 2 objects");
+    XCTAssertEqual([intObjects.firstObject intCol], 1);
+    XCTAssertEqual([intObjects.lastObject intCol], 2);
+}
+
+- (void)testRealmExitsAtPath {
+    // Create a seed realm with two objects
+    RLMRealmConfiguration *seedConfig = [RLMRealmConfiguration defaultConfiguration];
+    seedConfig.fileURL = RLMTestRealmURL();
+    RLMRealm *realm = [RLMRealm realmWithConfiguration:seedConfig error:nil];
+    [realm beginWriteTransaction];
+    [IntObject createInRealm: realm withValue:@[@1]];
+    [IntObject createInRealm: realm withValue:@[@2]];
+    [realm commitWriteTransaction];
+
+    NSURL *fileURL = [NSURL fileURLWithPath:RLMRealmPathForFile(@"filename.realm")];
+    assert(![[NSFileManager defaultManager] fileExistsAtPath:fileURL.path]);
+
+    // Create a realm at destination path; add one object
+    RLMRealmConfiguration *configuration = [RLMRealmConfiguration defaultConfiguration];
+    configuration.fileURL = fileURL;
+    {
```

Comment:
The objects here in this scope won't necessarily be released on exit of the scope. So use `@autorelesasepool`